explorer/app/apis/catalog/anvil-catalog/common/transformers.ts

```diff
@@ -25,11 +31,12 @@ export const getConsentCodes = (
 
 /**
  * Returns the consortium.
- * @param anvilCatalogEntity - AnVIL catalog entity.
  * @returns String value of consortium.
+ * @param anVILCatalogConsortium - the consortium.
```

Comment:
Revert lines 35 - 39. Each AnVIL entity (workspaces, consortium and studies) use this transformer.

Reply:
Does this look ok now?

Reply:
Perfect thank you!

Reply:
If so, can you resolve the conversation? Thanks! D